packages/itwin/tree-widget/src/test/trees/models-tree/internal/ModelsTreeVisibilityHandler.test.ts

```diff
@@ -2535,6 +2535,7 @@ describe("HierarchyBasedVisibilityHandler", () => {
               // FIXME: This is strange from the UX perspective
               groupingNode: ({ elementIds }) => (elementIds.includes(parentElementId) ? "visible" : "hidden"),
               element: ({ elementId }) => (elementId === parentElementId ? "visible" : "hidden"),
+              category: ({ categoryId }) => (categoryId === childCategoryId ? "hidden" : "visible"),
```

Comment:
flipped the comparison to stay consistent with other checks

```suggestion
              category: ({ categoryId }) => (categoryId === parentCategoryId ? "visible" : "hidden"),
```